source/globalCommands.py

```diff
@@ -1698,6 +1698,23 @@ def script_braille_toggleTether(self, gesture):
 	script_braille_toggleTether.__doc__ = _("Toggle tethering of braille between the focus and the review position")
 	script_braille_toggleTether.category=SCRCAT_BRAILLE
 
+	def script_braille_toggleFocusContextPresentation(self, gesture):
+		values = [x[0] for x in braille.focusContextPresentations]
+		labels = [x[1] for x in braille.focusContextPresentations]
+		try:
+			index = values.index(config.conf["braille"]["focusContextPresentation"])
+		except:
+			index=0
+		newIndex = index+1 if index+1<len(values) else 0

```

Comment:
FYI, this can be shortened to:

> `newIndex = index % len(values)`

I'll leave it up to you as to whether you want to change this. Whether it makes things more readable or not is certainly debatable. :)